plugins/techdocs-backend/src/techdocs/stages/prepare/helpers.ts

```diff
@@ -93,3 +93,46 @@ export const checkoutGitRepository = async (
 
   return repositoryTmpPath;
 };
+
+// Could be merged with checkoutGitRepository
+export const checkoutGithubRepository = async (
+  repoUrl: string,
+): Promise<string> => {
+  const parsedGitLocation = parseGitUrl(repoUrl);
+
+  // Should propably not be hardcoded names of env variables, but seems too hard to access config down here
+  const user = process.env.GITHUB_PRIVATE_TOKEN_USER || '';
+  const token = process.env.GITHUB_PRIVATE_TOKEN || '';
+
+  const repositoryTmpPath = path.join(
+    // fs.realpathSync fixes a problem with macOS returning a path that is a symlink
+    fs.realpathSync(os.tmpdir()),
+    'backstage-repo',
+    parsedGitLocation.source,
+    parsedGitLocation.owner,
+    parsedGitLocation.name,
+    parsedGitLocation.ref,
+  );
+
+  console.log(repoUrl);
```

Comment:
Please remove.

Reply:
Yerh ofc, forgot about that one :) https://github.com/spotify/backstage/pull/2247/commits/95121814e0a190ac97253930e83b20f8440497ad